app/src/main/java/com/example/songme/utils/Constant.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.songme.utils
+
+const val CONNECT_TIMEOUT = 5000
+const val READ_TIMEOUT = 5000
+const val FLAG_EMPTY = 113
```

Comment:
Để vào trong object cho dễ quản lý 
```suggestion
object Constants {
    const val CONNECT_TIMEOUT = 5000
    const val READ_TIMEOUT = 5000
    const val FLAG_EMPTY = 113
}
```